src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1205,13 +1206,6 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
 
     self->tx.available_quota = self->super.super.config.tx_qp_len -
                                ucs_min(self->super.super.config.tx_qp_len, config->quota);
-    /* Set max_iov for put_zcopy and get_zcopy */
-    uct_ib_iface_set_max_iov(&self->super.super.super,
```

Comment:
how about moving config.max_iov field to verbs transports only?

Reply:
i'll have either duplicate the value in ud and rc (and not use common get/set funcs) or introduce common struct with one field.

Reply:
either option is better than having an unused field in mlx5